app/controllers/sections_controller.rb

```diff
@@ -32,6 +31,7 @@ def show_events
                  @events.series.find_by(highlighted: 'true')
```

Comment:
Can you check this indentation, please?

Reply:
Also, can you check if Rubocop can check tab/spaces indentation standards?

Reply:
If I align that like this:
```
    @highlight = @events.videos.find_by(highlighted: 'true') ||
      @events.series.find_by(highlighted: 'true')
```
It complains about this:
```
app/controllers/sections_controller.rb:31:7: C: Layout/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.
      @events.series.find_by(highlighted: 'true')
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Docs: 
http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutmultilineoperationindentation
It doesn't mention the styleguide.

It would accept this:
```
    @highlight =
      @events.videos.find_by(highlighted: 'true') ||
      @events.series.find_by(highlighted: 'true')
```

The Layout section checks many things related to indentation.
http://rubocop.readthedocs.io/en/latest/cops_layout/#layout